addons/web/static/src/views/widgets/ribbon/ribbon.scss

```diff
@@ -38,7 +23,6 @@
 
     &-top-right {
         margin-top: -$o-sheet-vpadding;
-        right: 0;
 
         &::before, &::after {
             border-top-color: black;
```

Comment:
please remove hardcoded colors ;)
Prioritise utility-classes, if it's not possible (like in this case) use variables